test/render.test.ts

```diff
@@ -0,0 +1,104 @@
+import { expect } from 'chai'
+import { MessageBlock, wizardStatus } from '../src/render/functions'
+import { projectSummary } from '../src/fixtures/mock-projects'
+
+const unAuthNoProjMessage: MessageBlock[] = [
+  {
+    text: 'Welcome to the Affinidi Wizard',
+    styled: 'Welcome to the Affinidi Wizard',
+  },
+  {
+    text: 'You are not authenticated yet.',
+    styled: 'You are not authenticated yet.',
+  },
+  {
+    text: 'Active project: no active projects',
+    styled: 'Active project: no active projects',
+  },
+]
+const testUserEmail = 'test@email.com'
+const breadcrumbs: string[] = [
+  'project management',
+  'creating a new project',
+  'generate an application',
+]
+const authNoProjMessage: MessageBlock[] = [
+  {
+    text: 'Welcome to the Affinidi Wizard',
+    styled: 'Welcome to the Affinidi Wizard',
+  },
+  {
+    text: `You are authenticated as: ${testUserEmail}`,
+    styled: `You are authenticated as: ${testUserEmail}`,
+  },
+  {
+    text: 'Active project: no active projects',
+    styled: 'Active project: no active projects',
+  },
```

Comment:
can we consider refactoring a bit those messages in the render/function.ts file
or move all constants (text) into render/test.ts or something and use them in function.ts ?